1/hangman.py

```diff
@@ -0,0 +1,45 @@
+# Problem: the game is currently case-sensitive. A != a
+import getpass
+
+# Split the keyword into characters and put them into an array
+def split(keyword):
+    return [char for char in keyword]
+
+# Identify the number of guesses allowed
+def minimum(answer):
+    if len(answer) < 5:
+        return 5
+    else:
+        return len(answer) + 2
+
+# Check if the guessed letter matches something in the array, if so, replace the empty space in box
+def check_answer(guess):
+    for index, letter in enumerate(answer):
+        if letter == guess:
+            box[index] = guess
+
+# Ask P1 to input a string for P2 to guess
+keyword = getpass.getpass("Player 1: Please enter the word you want Player 2 to guess: ")
+answer = split(keyword)
+print ("Player 2: There are ",len(answer)," characters you need to guess")
+
+# Create a box to showcase P2 the empty places
+box = []
+while len(box) < len(answer):

```

Comment:
you can create box like so `box = ["_" for i in range(len(answer))]`

this means, you are creating box to be an array and you're initializing its values to be "_" for each position in a range of 0 to length of answer.